simplified_galaxy_stable_container_creation.sh

```diff
@@ -50,7 +56,11 @@ if [ -n $GALAXY_REPO ]
 	      DOCKERFILE_INIT_1=Dockerfile_init
        fi
        docker build $NO_CACHE --build-arg GALAXY_REPO=$GALAXY_REPO --build-arg GALAXY_RELEASE=$GALAXY_RELEASE --build-arg ISATOOLS_LITE_INSTALL=True -t $DOCKER_REPO$DOCKER_USER/galaxy-init-pheno:$TAG -f docker-galaxy-stable/compose/galaxy-init/$DOCKERFILE_INIT_1 docker-galaxy-stable/compose/galaxy-init/
-       docker push $DOCKER_REPO$DOCKER_USER/galaxy-init-pheno:$TAG
+       if [[ ! -z ${PUSH_INTERMEDIATE_IMAGES+x} ]];
+       then
+	   echo "Pushing intermediate image $DOCKER_REPO$DOCKER_USER/galaxy-init-pheno:$TAG"
```

Comment:
To be pedantic, it would be more defensive to put `$DOCKER_REPO$DOCKER_USER/galaxy-init-pheno:$TAG` in a variable, which you can reference in both lines 61 and 62.

Sooner or later someone's going to forget to update line 61 :-)